source/admin_manual/list_of_events.rst

```diff
@@ -2617,6 +2617,33 @@ metacache_user_clean_finished
 | error                 | Error message if the upload failed                   |
 +-----------------------+------------------------------------------------------+
 
+metacache_pull_started
+^^^^^^^^^^^^^^^^^^^^^^
+metacache_pull_finished
+^^^^^^^^^^^^^^^^^^^^^^^
+
++-----------------------+------------------------------------------------------+
+| Field                 | Description                                          |
++=======================+======================================================+
+| source_host           | From which host metacache is being pulled            |
++-----------------------+------------------------------------------------------+
+| type                  | "server" if the event is emitted by the host that    |
+|                       | metacache is being pulled from. (This host is        |
+|                       | "source_host")                                       |
+|                       |                                                      |
+|                       | "client" if the event is emitted by the host that    |
+|                       | is pulling metacache from "source_host"              |
```

Comment:
This still seems a bit confusing. When I run `doveadm metacache pull`, a "client" event is sent by doveadm process? But also on the remote doveadm-server side another "server" event is also sent?

Reply:
That is how it is currently implemented. I thought it would be nice to have both client and server events.